src/h5web/vis-packs/core/heatmap/ScalingMesh.tsx

```diff
@@ -0,0 +1,34 @@
+import { useCanvasScales } from '../hooks';
+import Mesh, { MeshProps } from './Mesh';
+
+interface Props extends MeshProps {
+  minAbscissa: number;
+  maxAbscissa: number;
+  minOrdinate: number;
+  maxOrdinate: number;
+}
+
+function ScalingMesh(props: Props) {
+  const { minAbscissa, maxAbscissa, minOrdinate, maxOrdinate, ...meshProps } =
+    props;
+  const { abscissaScale, ordinateScale } = useCanvasScales();
+
+  const left = abscissaScale(minAbscissa);
+  const right = abscissaScale(maxAbscissa);
+  const x = abscissaScale(minAbscissa + (maxAbscissa - minAbscissa) / 2);
+
+  const bottom = ordinateScale(minOrdinate);
+  const top = ordinateScale(maxOrdinate);
+  const y = ordinateScale(minOrdinate + (maxOrdinate - minOrdinate) / 2);
```

Comment:
I make use of the canvasScales to translate the limits coordinates (and the center) in the canvas coordinate system.